### PR TITLE
Refactor piped list components

### PR DESCRIPTION
**Pull request by cakecatz**
**What this PR does / why we need it**:

Create piped table row component and use it.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nghialv** [comment]
Thank you.
/lgtm

File: pkg/app/web/src/pages/settings/piped/index.tsx [hunk 1]
```diff
@@ -182,36 +159,14 @@ export const SettingsPipedPage: FC = memo(function SettingsPipedPage() {
...
-                    <IconButton
-                      edge="end"
-                      aria-label="open menu"
-                      onClick={(e) => handleMenuOpen(e, piped)}
-                    >
-                      <MoreVertIcon />
-                    </IconButton>
-                  </TableCell>
-                </TableRow>
+                <PipedTableRow
```
Comment (khanhtc1202) [inline pkg/app/web/src/pages/settings/piped/index.tsx:162]: nits, prefix `Piped` is redundant to me 👀

Comment (cakecatz) [reply pkg/app/web/src/pages/settings/piped/index.tsx:162]: @khanhtc1202 
If there were no `Piped`, wouldn't it be hard to distinguish it from the `TableRow` in `material-ui`?

Comment (khanhtc1202) [reply pkg/app/web/src/pages/settings/piped/index.tsx:162]: ops, I see 👀 In that case, maybe my suggestion is: rename the component to `TableRow` and in the calling side
```typescript
import { TableRow as PipedTableRow } from "./piped-table-row";
```
but in the second thought, It is quite laborious to do so, so the current implementation is LGTM 👍 
I'm going to merge this PR 🏃‍♂️

**Comment by khanhtc1202** [comment]
Thank you 🚀 
/approve

**Pull request merged**